CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [Unreleased]
 ------------
 
+[v0.5.0] - 2019-4-9
```

Comment:
A couple of additional fixups:

* Anand's note for the fix of the CVE should be moved under the **Fixed** section instead of **Added**

* There's also a typo in his text for "... will now write to the supplied argument if it _lies_ in host memory."

* In the **Changed** section, there's an outdated link regarding the QE changes: "To resolve this issue, please install the latest version of the [Intel SGX DCAP packages](https://01.org/intel-software-guard-extensions/downloads) (1.0.1 or newer) on the system that generates the remote report.~~, which as of the time of this change is version 1.0.1 and can be found here: **https://download.01.org/intel-sgx/dcap-1.0.1/dcap_installer/ubuntuServer1604/**~~"

* Mark both the v0.4.0 and v0.4.1 versions as `[DEPRECATED]` since we're not yanking them, but folks should be aware they should not be using them anymore.

* Folks haven't been correctly updating the commit ranges for each update. If you look at the bottom, you'll see that the tagging for the versions in the titles are intended to be links:

    `[Unreleased]: https://github.com/microsoft/openenclave/compare/v0.4.0...HEAD`

So we should have:
```
  [Unreleased]: https://github.com/microsoft/openenclave/compare/v0.5.0...HEAD
  [v0.5.0] https://github.com/microsoft/openenclave/compare/v0.4.1...v0.5.0
  [v0.4.1] https://github.com/microsoft/openenclave/compare/v0.4.0...0.4.1
```

Reply:
@CodeMonkeyLeet - Will you take another look to see if addressed all of these issues?